dev/com.ibm.websphere.appserver.features/test/src/com/ibm/ws/feature/tests/VersionlessTest.java

```diff
@@ -391,17 +408,29 @@ private static void put(Map<String, List<String>> storage, String... data) {
         put(useCohorts, "microProfile",
             "com.ibm.websphere.appserver.microProfile", "io.openliberty.microProfile");
 
-        put(useCohorts, "webProfile",
```

Comment:
Why do we want to remove this? There are some features (ex. servlet) that exist only in webProfile not in javaee/jakartaee convenience features but are part of the jakartaee/javaee platforms. Could we possibly leave this in and then for any features in webProfile we don't want we can add to the "skipFeatures" list below?

Reply:
jakartaee and javaee features both include webprofile feature.  With the changes all features are examined now in javaee and jakartaee where it was only looking at the first layer before and was missing things like jaxrsClient / restfulWSClient / mdb / ejbRemote / ejbPersistentTimer, etc.  So it is redundant to have webprofile since we will look at webprofile with javaee / jakartaee now with the changes done in the test.